classes/reports/realtime/cli.php

```diff
@@ -91,7 +91,7 @@ public function __construct()
 
         $runnerResultField = new runner\result\cli();
         $runnerResultField
-            ->setSuccessColorizer(new colorizer('0;37', '42'))
+            ->setSuccessColorizer(new colorizer('0;97', '42'))
```

Comment:
I agree... What about to choose 1;37 then ?
I'm not used to term colors...
Thank you